<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Rietesh" failures="3" tests="20" name="Test" time="213.978" errors="0" timestamp="2022-05-22T16:45:59 IST">
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="12.497"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="7.761"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="8.866"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="12.979"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="18.844"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="15.828"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="10.706"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="11.186"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="10.242"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="9.572"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="8.882"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="8.583">
    <failure type="org.openqa.selenium.NoSuchElementException" message="Unable to find element with locator By.xpath: //*[@id=&amp;quot;pass&amp;quot;]
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: &amp;apos;4.1.2&amp;apos;, revision: &amp;apos;9a5a329c5a&amp;apos;
System info: host: &amp;apos;RIETESH&amp;apos;, ip: &amp;apos;192.168.42.83&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.1&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to find element with locator By.xpath: //*[@id="pass"]
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.1.2', revision: '9a5a329c5a'
System info: host: 'RIETESH', ip: '192.168.42.83', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.1'
Driver info: driver.version: RemoteWebDriver
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:165)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:382)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:374)
at dataDrivenTesting.DataProvider.testcases(DataProvider.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testcases -->
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="10.32"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="9.314"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="13.441"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="13.116"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="10.898"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="10.783"/>
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="2.245">
    <failure type="org.openqa.selenium.WebDriverException" message="unknown error: cannot determine loading status
from disconnected: received Inspector.detached event
  (Session info: chrome=101.0.4951.67)
Build info: version: &amp;apos;4.1.2&amp;apos;, revision: &amp;apos;9a5a329c5a&amp;apos;
System info: host: &amp;apos;RIETESH&amp;apos;, ip: &amp;apos;192.168.42.83&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [7a535028d3b56701c6675183948080a4, get {url=https://www.facebook.com/login/}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 101.0.4951.67, chrome: {chromedriverVersion: 101.0.4951.41 (93c720db8323..., userDataDir: C:\Users\rites\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:56746}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:56746/devtoo..., se:cdpVersion: 101.0.4951.67, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 7a535028d3b56701c6675183948080a4">
      <![CDATA[org.openqa.selenium.WebDriverException: unknown error: cannot determine loading status
from disconnected: received Inspector.detached event
  (Session info: chrome=101.0.4951.67)
Build info: version: '4.1.2', revision: '9a5a329c5a'
System info: host: 'RIETESH', ip: '192.168.42.83', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [7a535028d3b56701c6675183948080a4, get {url=https://www.facebook.com/login/}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 101.0.4951.67, chrome: {chromedriverVersion: 101.0.4951.41 (93c720db8323..., userDataDir: C:\Users\rites\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:56746}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:56746/devtoo..., se:cdpVersion: 101.0.4951.67, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: 7a535028d3b56701c6675183948080a4
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:558)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:327)
at dataDrivenTesting.DataProvider.testcases(DataProvider.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testcases -->
  <testcase classname="dataDrivenTesting.DataProvider" name="testcases" time="6.455">
    <failure type="java.lang.IllegalArgumentException" message="Keys to send should be a not null CharSequence">
      <![CDATA[java.lang.IllegalArgumentException: Keys to send should be a not null CharSequence
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:92)
at dataDrivenTesting.DataProvider.testcases(DataProvider.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- testcases -->
</testsuite> <!-- Test -->
